src/utils/errors.ts

```diff
@@ -77,6 +78,25 @@ export class WebhookValidationException extends HttpException {
   }
 }
 
+export class NotImplementedException extends HttpException {
+  constructor(message?: string) {
+    super(HttpStatus.NOT_IMPLEMENTED, `Not implemented. ${message}`);
+  }
+}
+
+export class WebhooksProcessingException extends Error {
+  webhookSource: WebhookSourceDTO;
+  payload: any;
```

Comment:
Why is this any? Is it possible to type the payload?

Reply:
Fixed.

Reply:
Fixed.